android/src/main/java/org/reactnative/camera/tasks/BarCodeScannerAsyncTask.java

```diff
@@ -41,14 +41,27 @@ protected Result doInBackground(Void... ignored) {
       BinaryBitmap bitmap = generateBitmapFromImageData(mImageData, mWidth, mHeight);
       result = mMultiFormatReader.decodeWithState(bitmap);
     } catch (NotFoundException e) {
-      // No barcode found, result is already null.
+      BinaryBitmap bitmap = generateBitmapFromImageData(rotateImage(mImageData,mWidth, mHeight),mHeight,mWidth);
+      try {
+        result = mMultiFormatReader.decodeWithState(bitmap);
+      } catch (NotFoundException e1) {
+        //no barcode Found
+      }
     } catch (Throwable t) {
       t.printStackTrace();
     }
 
     return result;
   }
-
+  private byte[] rotateImage(byte[]imageData,int width, int height) {
```

Comment:
this is not a 1:1 copy from the old method.

Why the changes?

Reply:
Before it was:
`rotated[x * width + width - y - 1] = imageData[x + y * height];`

Reply:
Ok, looking again, the difference is because before y and x were `height` and `width` and now they are `width` and `height`. If it produces the same result, ok hehe

Reply:
Yes on last PR i swaped the naming, but now i just copied from RCTCamera and yes it produce the same :-D